crates/sui-network/src/state_sync/mod.rs

```diff
@@ -1316,6 +1316,7 @@ where
         tokio::time::sleep(duration).await;
         return Err(checkpoint);
     };
+    debug!("completed checkpoint contents sync");
```

Comment:
Adding a checkpoint id would be better

Reply:
checkpoint sequence number is already part of the log message because of the instrument annotation on the fn